drain.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Ansible playbook to drain Slurm compute nodes. Waits for compute nodes to be
```

Comment:
Technically it's not a playbook because it doesn't select hosts or groups on which to apply the tasks.  But that's fine, just me being pedantic...

Reply:
You're right, I noticed it as I was writing the comment but only researched it later. In a role this would be a task file. I created a `tasks` directory for them.